packages/hashi/README.md

```diff
@@ -22,3 +22,104 @@ Step 1: Install Hashi package deps (run from Kolibri root)
 Step 2: Build Kolibri and hashi
 
 `yarn run build`
+
+
+
+Custom Channels
```

Comment:
We should call this 'custom navigation' as this can happen in any topic, not just at the channel level.